packages/mdc-button/mdc-button.scss

```diff
@@ -29,7 +29,7 @@
   @include mdc-button-ink-color(text-primary-on-light);
   @include mdc-button-ripple((base-color: black, opacity: .16));
 
-  @include mdc-theme-dark(".mdc-button") {
+  @include mdc-theme-dark(".mdc-button", true /* compound */) {
```

Comment:
I dont think this one is compound....this will end up creating a selection like `.mdc-button--theme-dark.mdc-button`. I think in this case you just need to write `@include mdc-theme-dark` with no arguments.